index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sign-Up Form</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/06aa7828ae.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <main>
+        <header>
+            <div class="account">
+                <div class="acc">Already have an Inuit account? <br> <a class="acc-link" href="#">I forgot my User ID or
+                        Password</a></div>
+
+                <div class="ol"><i class="fas fa-lock"></i> Sign In</div>
+            </div>
+        </header>
+        <section>
+            <div class="logo"><img src="/image/logo.png" alt="logo"></div>
+            <div class="center">
+                <div class="center-top">
+                    <h3>Create an Intuit account</h3>
+                    <h5>One account for everything Intuit, including Mint.<br><a href="#">Learn more</a></h5>
+
+                </div>
+                <form action="POST">
+                    <div class="insid"><label for="email">Email address</label>
+                        <input class="input-field" type="text" id="email"></div>
+                    <div class="insidee">
+                        <label for="phone">Phone (recommended)</label>
+                        <input class="input-field" type="number" id="phone">
+
+                    </div>
+                    <div class="insides">
+                        Standard call, messaging or data rates may apply.
+                    </div>
+                    <div class="inside">
+                        <label for="password">Password</label>
+                        <div class="lock">
+                            <input class="input-field ccc" type="password" id="password" placeholder=""> <span><i
+                                    class="fas fa-lock-open"></i></span>
+                        </div>
+                    </div>
+                    <button class="done" type="submit"><i class="fas fa-lock"></i> Create Account</button>
+                </form>
```

Comment:
- The email should have an `<input>` of type `email`.

- The phone should have an `<input>` of type` tel`.